src/api/list-entries.ts

```diff
@@ -0,0 +1,35 @@
+import { fetchApiJson, fetchApiText } from ".";
+
+export interface ConfiguredEntry {
+  name: string;
+  filename: string;
+  loaded_integrations: string[];
+  deployed_version: string;
+  current_version: string;
+  path: string;
+  comment: string;
+  address: string;
+  target_platform: string;
+  online_status: boolean;
+}
+
+export interface ImportableEntry {
+  name: string;
+  import_config: string;
+  project_name: string;
+  project_version: string;
+}
+
+export interface ListEntriesResult {
+  configured: ConfiguredEntry[];
+  importable: ImportableEntry[];
+}
+
+export const getDashboardEntries = () =>
```

Comment:
Let's call it `getEntries`. It's up to the calling method to decide how to show it.